app/src/main/java/ru/netology/fmhandroid/adapter/PatientListAdapter.kt

```diff
@@ -35,7 +36,18 @@ class PatientListAdapter(
         fun bind(patient: Patient) {
             binding.apply {
                 (patient.lastName + " " + patient.firstName + " " + patient.middleName).also { patientName.text = it }
-                patientLocation.text = if (patient.inHospice) "Да" else "Нет"
+                patientLocation.text =
+                    when (patient.status) {
+                        PatientStatusEnum.ACTIVE -> {
+                            itemView.context.getString(R.string.patients_status_active)
+                        }
+                        PatientStatusEnum.EXPECTED -> {
+                            itemView.context.getString(R.string.patients_status_expected)
+                        }
+                        PatientStatusEnum.DISCHARGED -> {
+                            itemView.context.getString(R.string.patients_status_discharged)
+                        }
+                    }
```

Comment:
Поменьше повторений в коде 😊
```suggestion
                val patientStatusResId = when (patient.status) {
                    PatientStatusEnum.ACTIVE -> R.string.patients_status_active
                    PatientStatusEnum.EXPECTED -> R.string.patients_status_expected
                    PatientStatusEnum.DISCHARGED -> R.string.patients_status_discharged
                }
                patientLocation.text = itemView.resources.getString(patientStatusResId )
```

Reply:
Так гораздо лучше! Исправлено.